packages/blade/src/components/SideNav/_decisions/decisions.md

```diff
@@ -0,0 +1,418 @@
+| titleSuffix | Slot after the title to add Badge, Counter                                                                                   | JSX                                                                 |                                                              |
+| ref         | Forwards the given ref. This also allows it to be a trigger for Tooltip                                                      | idk man ref types are hard.<br/> I'll figure out while implementing |                                                              |
+| children    | SideNavLink children slot. Items inside children turn into next level item with parent as a trigger                          | JSX                                                                 | undefined                                                    |
+
+#### Example Nav Links
+
+<table>
+
+<tr><td>Code</td><td>Preview</td></tr>
+
+<tr>
+<td>
+
+<!-- prettier-ignore -->
+```jsx
+<SideNavLink
+  icon={BillIcon}
+  title="Vendor Payments"
+  href="/vendor-payments"
+/>
+```
+
+</td>
+<td><img src="navlink-eg-1.png" width="200px" /></td>
+</tr>
+
+<tr>
+<td>
+
+<!-- prettier-ignore -->
+```jsx
+<SideNavLink
+  icon={PayoutIcon}
+  title="Create Payouts"
+  href="/payouts"
+  trailing={
+    <Button
+      href="/payouts/create"
+      icon={PlusIcon}
+      size="xsmall"
+      variant="tertiary"
+    />
+  }
+/>
+```
+
+</td>
+<td><img src="navlink-eg-2.png" width="200px" /></td>
+</tr>
+
+<tr>
+<td>
+
+<!-- prettier-ignore -->
+```jsx
+<SideNavLink
+  icon={LayoutIcon}
+  title="L1 Item Name"
+  href="/new-item-link"
+  titleSuffix={
+    <Badge color="positive">New</Badge>
+  }
+/>
+```
+
+</td>
+<td><img src="navlink-eg-3.png" width="200px" /></td>
+</tr>
+
+<tr>
+
+<td>
+
+<!-- prettier-ignore -->
+```jsx
+<Tooltip content="Action Name (Cmd + P)">
+  <SideNavLink
+    icon={LayoutIcon}
+    title="L1 Item Name"
+    href="/new-item-link"
+  />
+</Tooltip>
+```
+
+</td>
+<td><img src="navlink-eg-4.png" width="240px" /></td>
+</tr>
+
+</table>
+
+### SideNavLevel
+
+Nested SideNavLevel components create new levels. This can be used to create L1 - L2 - L3 levels in your navbar
+
+| **Props** | **Description**                                       | **Type** | **Default Value** |
+| --------- | ----------------------------------------------------- | -------- | ----------------- |
+| title     | Only applicable in L2. Becomes the title of the Level | string   |                   |
+| children  | children slot. Accepts SideNavLink as children        | JSX      |                   |
+
+<table>
+<tr>
+<td>
+
+```jsx
+<SideNav>
+  {/* L1 Items */}
+  <SideNavLink title="L1 Item" />
+
+  <SideNavLink title="L2 Trigger">
```

Comment:
what if we just have NavLink and provide `onClick` and `href` as 2 props and if devs want nested levels they can use onClick and if its href that's a navigation? its the same terminology as link as button and link as nav

Reply:
Currently clicking item also changes the route so need href there